streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -133,112 +129,179 @@
 
 
 
+    public static final String SECURITY_PROTOCOL_CONFIG = CommonClientConfigs.SECURITY_PROTOCOL_CONFIG;
```

Comment:
I don't think we should add these configs to the public interface. Developers can already add Consumer & Producer Configs to the StreamsConfig - they both have all of the configs defined here and we should just use those. For example have a look at `StreamsConfig.getProducerConfigs(...)`